agent-core/src/main/java/com/newrelic/agent/android/instrumentation/TransactionState.java

```diff
@@ -46,6 +47,16 @@ enum State {
     private TransactionData transactionData;
     private TraceContext trace;
 
+    private Map<String, String> params = null;
```

Comment:
As mentioned, the container can be initialized here, or allocated in the ctor, and then avoid null checks when accessing it.

Reply:
i will do it.